docs/root/configuration/http/http_filters/router_filter.rst

```diff
@@ -413,6 +425,8 @@ statistics:
   upstream_rq_<\*xx>, Counter, "Aggregate HTTP response codes (e.g., 2xx, 3xx, etc.)"
   upstream_rq_<\*>, Counter, "Specific HTTP response codes (e.g., 201, 302, etc.)"
   upstream_rq_retry, Counter, Total request retries
+  upstream_rq_retry_backoff_exponential, Counter, Total retries using the exponential backoff strategy
+  upstream_rq_retry_backoff_ratelimited, Counter, Total retries using the ratelimited backoff strategy
```

Comment:
I think we can skip this on the virtual cluster. I doubt this will be used practically.

Reply:
Awesome, removed it.